config.json

```diff
@@ -30,6 +30,18 @@
         "logic"
       ]
     },
+    {
+      "slug": "reverse-string",
+      "uuid": "553a6be7-eecb-45dc-9cea-05126c525f1b",
```

Comment:
Looks like you'll need to change the UUID.  Travis says they need to be unique _across tracks_, so the build is failing:

> -> The following UUID was found in multiple Exercism tracks. Each exercise UUID must be unique across tracks.